# TODO (custom-display-name/display-name-only-exported-component)

TODO: Why this rule is useful.

## ğŸ“– Rule Details

TODO: How this rule will report code.

ğŸ‘ Examples of **correct** code for this rule:

```ts

```

ğŸ‘ Examples of **incorrect** code for this rule:

```ts

```
