# TODO (custom-display-name/display-name-only-exported-component)

TODO: Why this rule is useful.

## 📖 Rule Details

TODO: How this rule will report code.

👍 Examples of **correct** code for this rule:

```ts

```

👎 Examples of **incorrect** code for this rule:

```ts

```
