# Exported Component requires displayName (custom-display-name/display-name-only-exported-component)

This rule prevents components which doesn't set `displayName` from exporting.

## ğŸ“– Rule Details

ğŸ‘ Examples of **correct** code for this rule:

```ts

```

ğŸ‘ Examples of **incorrect** code for this rule:

```ts

```
