# Exported Component requires displayName (custom-display-name/display-name-only-exported-component)

This rule prevents components which doesn't set `displayName` from exporting.

## 📖 Rule Details

👍 Examples of **correct** code for this rule:

```ts

```

👎 Examples of **incorrect** code for this rule:

```ts

```
